lib/animina/accounts/resources/photo.ex

```diff
@@ -311,9 +315,10 @@ defmodule Animina.Accounts.Photo do
 
   def create_photo_flags(record) do
     if File.exists?("priv/static/uploads/" <> record.filename) do
-      {flags, description} = ImageTagging.tag_image_using_llava("#{record.filename}")
+      {flags, description} =
+        ImageTagging.tag_image_using_llava("#{record.filename}")
```

Comment:
I don't like the function name "tag_image_useing_llava". Mainly because it hs `llava` hardcoded in it. We can change the LLM any day and we will (probably within 12 months). Keep it generic. Something like `auto_tag_image`.